tests/ConfigTest.php

```diff
@@ -25,6 +26,47 @@
  */
 final class ConfigTest extends \PHPUnit_Framework_TestCase
 {
+    public function testConfigRulesUsingSeparateMethod()
+    {
+        $config = new Config();
+        $configResolver = new ConfigurationResolver(
+            $config, array(
+                'rules' => 'cast_spaces,braces',
+            ),
+            getcwd()
+        );
+        $fixers = $configResolver->getFixers();
+        $this->assertInstanceOf(\PhpCsFixer\Fixer\CastNotation\CastSpacesFixer::class, $fixers[0]);
+        $this->assertInstanceOf(\PhpCsFixer\Fixer\Basic\BracesFixer::class, $fixers[1]);
+    }
+
+    public function testConfigRulesUsingJsonMethod()
+    {
+        $config = new Config();
+        $configResolver = new ConfigurationResolver(
+            $config, array(
+                'rules' => '{"array_syntax": {"syntax": "short"}, "cast_spaces": true}',
+            ),
+            getcwd()
+        );
+        $fixers = $configResolver->getFixers();
+        $this->assertInstanceOf(\PhpCsFixer\Fixer\ArrayNotation\ArraySyntaxFixer::class, $fixers[0]);
+        $this->assertInstanceOf(\PhpCsFixer\Fixer\CastNotation\CastSpacesFixer::class, $fixers[1]);
```

Comment:
* please do not rely on the order here, `getFixers` may return fixers in different order depends on priority of the fixers
* please check if the config was applied